resources/idlharness.js

```diff
@@ -217,7 +239,7 @@ IdlArray.prototype.internal_add_idls = function(parsed_idls, options)
 
     if (options && options.only && options.except)
     {
-        throw "The only and except options can't be used together."
+        throw new IdlHarnessError("The only and except options can't be used together.");
```

Comment:
See comment elsewhere about doing the message comparison more generically. That'd make it possible to just use `new Error` everywhere in idlharness.js.

Reply:
Keeping explicit error type (I see value in it) but made sure not to ref it from `testharness.js`